File: JavaScript/bee 1046.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split(/\s+/);

let [horaInicial, horaFinal] = lines.map(Number)

if(horaFinal > horaInicial){
    console.log(`O JOGO DUROU ${horaFinal - horaInicial} HORA(S)`)
}
else{ // exemplo se for entre 22h e 1h: (24 - 22) + 1 == 3; pois a hora inicial não pode ser mais tarde do que a final
    console.log(`O JOGO DUROU ${(24 - horaInicial) + horaFinal} HORA(S)`)
}

File: Python/bee 1046.py
import math

horaInicial, minutoInicial, horaFinal, minutoFinal = map(int, input().split(" "))

inicio = 60 * horaInicial + minutoInicial
fim = 60 * horaFinal + minutoFinal

duracao = fim - inicio

if duracao <= 0:
    duracao += 1440

horas = math.floor(duracao/60)
minutos = duracao % 60 

print(f"O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)")